packages/admin/admin/src/common/buttons/Button.tsx

```diff
@@ -107,14 +108,16 @@ export const Button = forwardRef(<C extends ElementType = "button">(inProps: But
         return (
             <MobileTooltip title={children} {...slotProps?.mobileTooltip}>
                 <span>
-                    <Root {...commonButtonProps}>{mobileIconNode}</Root>
+                    <Root disableFocusRipple disableTouchRipple {...commonButtonProps}>
```

Comment:
Please add these props to `commonButtonProps` so they only need to be defined once, like the other props.